ui/tests/helpers/module-for-job.js

```diff
@@ -143,6 +170,23 @@ export function moduleForJobWithClientStatus(title, jobFactory, additionalTests)
       }
     });
 
+    test('job status summary is collapsed when not authorized', async function(assert) {
+      const clientToken = server.create('token', { type: 'client' });
+      await Tokens.visit();
+      await Tokens.secret(clientToken.secretId).submit();
```

Comment:
I think that we can't set a token directly into `localStorage` after visiting a page? I'm not sure if I missing something, but it didn't work in a few acceptance tests, so I had to do it this way.

And TBH I kind of like it, feels more intentional 😅

Reply:
We need to establish boundaries for API's we don't control. Ideally, we should wrap localStorage access into a service and then write mocks.

Reply:
This is a bug. We initialize query parameters on `period` and `parameterized` jobs whereas we don't follow this behavior for other jobs. Will need to take a deeper dive into this.